llvm/test/Analysis/LoopAccessAnalysis/wrapping-pointer-versioning.ll

```diff
@@ -1,6 +1,6 @@
-; RUN: opt -passes='print<access-info>' -aa-pipeline='basic-aa' -disable-output < %s  2>&1 | FileCheck %s --check-prefix=LAA
-
-target datalayout = "e-m:o-i64:64-f80:128-n8:16:32:64-S128"
+; NOTE: Assertions have been autogenerated by utils/update_analyze_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -passes='print<access-info>' -aa-pipeline='basic-aa' \
```

Comment:
not sure if `-aa-pipeline='basic-aa'` is needed, if not might be worth simplifying while you are modifying the run line here

Reply:
It is needed, unfortunately.

Reply:
Oh interesting